src/solver/objects.h

```diff
@@ -634,6 +639,7 @@ typedef struct
    char*         ID;              // link ID
    int           type;            // link type code
    int           subIndex;        // index of link's sub-category
+   int*          extPollutFlag;   // external pollutant flag
```

Comment:
why is this a pointer?

Reply:
isn't it just 1 or 0 for state?

Reply:
You are right it is 0 or 1. But given that it is set by an external function by reference, we had to define it as a pointer.